gitlab_runner/manifest.json

```diff
@@ -16,8 +17,8 @@
     "source control",
     "issue tracking"
   ],
-  "doc_link": "https://docs.datadoghq.com/integrations/gitlab_runner/",
+  "metric_prefix": "gitlab_runner.", // is this valid see metadata file?
```

Comment:
The metadata file for this intergration doesn't seem correct... Keeping the prefix to `gitlab_runner.` we don't have gitlab dashboards